modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -409,3 +409,24 @@ If you are using a reverse proxy like Traefik and the reverse proxy manages the
 == Manage Unfinished Uploads
 
 See xref:deployment/services/storage-users.adoc#manage-unfinished-uploads[Manage Unfinished Uploads] at the _Storage Users_ service for details. 
+
+== Reindex a Space for Search
+
+In rare cirumstances, it can happen that it is necessary to initiate reindexing manually for a given space or user. Though the search service handles exeption cases automatically, it can happen that the search service was not able to complete indexing due to a dirty shut down of the service or because of a bug. The following command should *only* be run on explicit request and supervision of ownCloud support. Note that behaviour and options may change without special notice. 
```

Comment:
```suggestion
In rare circumstances, it can be necessary to initiate indexing manually for a given space or user. Though the search service handles exception cases automatically, it can happen that the search service was not able to complete indexing due to a dirty shut down of the service or because of a bug. The following command should *only* be run on explicit request and supervision of ownCloud support. Note that behavior and options may change without special notice. 
```